### PR TITLE
add term UBERON:8000008 cementocyte lacuna

### LINKED ISSUE #1397: NTR: cementocyte lacuna
NTR: cementocyte lacuna
Suggested parent: UBERON_0001753  cementum _(I think the cemetocyte lacuna is part of the cementum, so maybe this is not the appropriate parent class?)_
Suggested def: The small cavity within the cellular cementum that is occupied by a cementocyte cell body.
xref: PMID:27029548, MP:0030555

related ticket: https://github.com/obophenotype/mammalian-phenotype-ontology/issues/2664

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
close #1397

### CODE REVIEW

File: src/ontology/uberon-edit.obo [hunk 1]
```diff
@@ -224754,6 +224754,13 @@ intersection_of: UBERON:0001137 ! dorsum
 intersection_of: in_right_side_of UBERON:0000468 ! multicellular organism
 created_by: http://orcid.org/0000-0001-5208-3432
 
+[Term]
+id: UBERON:8000008
+name: cementocyte lacuna
+def: "The small cavity within the cellular cementum that is occupied by a cementocyte cell body." [MP:0030555, PMID:27029548]
+relationship: part_of UBERON:0001753 ! cementum
```
Comment (cmungall) [inline src/ontology/uberon-edit.obo:224761]: no isa

**Review by cmungall — CHANGES_REQUESTED** [review]

**Comment by cmungall** [comment]
not a blocker, but when doing something like this, you should be thinking: I see a CL class mentioned in the text? What is the relationship between the cell and the structure? part-of? located-in? has-soma-located-in? 

Also, are there commonalities between cementoblast-cementum lacunae relationships and the relationships between osteoblasts and lacunae in bones?

it can help future maintenance of the ontology tremendously if things are viewed holistically like this. otherwise we will have one curator making statements about one set of structures, and another about a homologous set of structures, both correct, but different perspectives, leading it confusion and inconsistency

**Comment by nicolevasilevsky** [comment]
should we add a new term for lacuna?
https://en.wikipedia.org/wiki/Lacuna_(histology)

File: src/ontology/uberon-edit.obo (hunk 1 above)
Comment (nicolevasilevsky) [reply src/ontology/uberon-edit.obo:224761]: added is_a 'anatomical cavity'

**Comment by nicolevasilevsky** [comment]
I think a cementoblast is located in the cementocyte lacuna. Would this be added to Uberon or CL?

From [wikipedia](https://en.wikipedia.org/wiki/Cementoblast):
Thus cementoblasts resemble bone-forming osteoblasts but differ functionally and histologically. The cells of cementum are the entrapped cementoblasts, the cementocytes. **Each cementocyte lies in its lacuna (plural, lacunae),** similar to the pattern noted in bone. These lacunae also have canaliculi or canals. Unlike those in bone, however, these canals in cementum do not contain nerves, nor do they radiate outward. Instead, the canals are oriented toward the periodontal ligament (PDL) and contain cementocytic processes that exist to diffuse nutrients from the ligament because it is vascularized.[3] The progenitor cells also found in the PDL region contribute to the mineralization of the tissue.[4]

**Comment by nicolevasilevsky** [comment]
> Also, are there commonalities between cementoblast-cementum lacunae relationships and the relationships between osteoblasts and lacunae in bones?

From what I just read, yes, I think this relationship is the same. We don't have the term lacunae in Uberon, but this is described in the definition of 'bone tissue':

Skeletal tissue with a collagen-rich extracellular matrix vascularized, mineralized with hydroxyapatite and typically including osteocytes located in lacunae that communicate with one another by cell processes (in canaliculi). Bone is deposited by osteoblasts.

**Review by cmungall — APPROVED** [review]

**Pull request merged**